nbbuild/external/binaries-list

```diff
@@ -17,3 +17,4 @@
 A7674A6D78B7FEA58AF76B357DAE6EA5E3FDFBE9 apitest-70.jar
 16398550402B27F81CD0D508CEF54B3E47A4A6DA org.apache.rat:apache-rat:0.12
 AB396EE119BFAD809CC9F09950CC22E2BCE2FE35 langtools-9.zip
+E32B3483FBF362C92088CB79E9F1F161F3F64A21 org.apidesign.javadoc:codesnippet-doclet:0.30
```

Comment:
-1

According to https://search.maven.org/artifact/org.apidesign.javadoc/codesnippet-doclet/0.31/jar this is GPL 3:

> Codesnippet Javadoc Doclet
>     Copyright (C) 2015-2018 Jaroslav Tulach - jaroslav.tulach@apidesign.org
> 
>     This program is free software: you can redistribute it and/or modify
>     it under the terms of the GNU General Public License as published by
>     the Free Software Foundation, version 3.0 of the License.

Reply:
It is a build time only dependency. The codesnippet is used when generating Javadoc.